src/test/java/uk/co/ramp/covid/simulation/ModelTest.java

```diff
@@ -38,10 +39,10 @@ public void setupParams() {
 
     @Test
     public void testBaseLine() {
-
+        final int inf = nInfections * 5;

```

Comment:
This looks strange.  @BlairArchibald could you explain why you changed this?

Reply:
For low numbers of infections you can get into cases where, for example, "more" adults die than pensioners (especially when some are now reasonably well protected in care homes with PPE etc) -- although it's really 1 adult death to 0 pensioners deaths. 

Cranking this up a bit just makes it a bit more robust to these small deltas.